.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,25 @@
+name: Integration Test
+
+on:
+  workflow_run:
+      workflows: ["DB12 Testing"]
+      types: 
+        - completed
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:  
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+    
+    - uses: dawidd6/action-download-artifact@v2
```

Comment:
What 's this for?

Reply:
We wanted to make sure that python2 scores are ~ equal to python3 scores so:
- we run db12 in `python2-3-app.yml` with each version of python (2.7, 3.6 and 3.9)
- we put the score in a file called `result_python<version>.txt` and we upload it in the artifacts
- then in `integration-test.yml` we download scores from the artifacts and we just compare the scores to make sure they are ~ equal.

We assumed that machines used to run the workflows are ~ identical but we observed differences in the scores.
So either machines are not identical and not identically used (probably the case), or there is a significant difference between python2 and python3 results.
See: https://github.com/DIRACGrid/DB12/pull/2#pullrequestreview-704832794

We should probably run `db12` with different versions of python directly from the `integration_test.yml` so that they run in the same environment.

Reply:
To make sure that we are using the same machine every time, and that such machine is idle, there's only one solution, and it's to use our owns runners. This can be done in GitHubActions (https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners) but for the moment it would probably be easier to use Jenkins.